app/models/allocation_version.rb

```diff
@@ -93,6 +95,10 @@ def self.deallocate_primary_pom(nomis_staff_id)
       )
   end
 
+  def self.offender_movement_type(movement_type)
```

Comment:
I don't think we'll need this at all if instead of passing in ADM or REL (which is info this class shouldn't need to know) the movement_service instead passed in AllocationVersion::OFFENDER_TRANSFERRED or AllocationVersion.OFFENDER_RELEASED.

Reply:
Of course! Refactor on it's way.